config/scheduler-chromeos.yaml

```diff
@@ -187,6 +187,14 @@ scheduler:
       result: pass
     platforms: *mediatek-platforms
 
+  - job: kselftest-discoverable-devices-probe
```

Comment:
Is `discoverable-devices-probe` a kselftest? If not, why the same naming layout is being used?

Reply:
It is a kselftest: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=dacf1d7a78bf8a131346c47bfba7fe1f3ff44beb